numpy/core/fromnumeric.py

```diff
@@ -2530,19 +2504,7 @@ def prod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     True
 
     """
-    kwargs = {}
-    if keepdims is not np._NoValue:
-        kwargs['keepdims'] = keepdims
-    if type(a) is not mu.ndarray:
-        try:
-            prod = a.prod
-        except AttributeError:
-            pass
-        else:
-            return prod(axis=axis, dtype=dtype, out=out, **kwargs)
-
-    return _methods._prod(a, axis=axis, dtype=dtype,
-                          out=out, **kwargs)
+    return _wrapreduction(a, np.multiply, 'prod', axis, dtype, out, keepdims=keepdims)
```

Comment:
For consistency either this should be `um.multiply` or the other ones should be `np.minimum` etc. I'd be inclined to go with the `np.` form for clarity

Reply:
Done!

Reply:
Done!